### DISCUSSION TITLE
Use sntrup761  in oqs provider

### DISCUSSION THREAD

**Discussion by seschu** [open-quantum-safe/liboqs#1657] [Q&A]
Hi there,
I am wondering whether it is possible to use sntrup761 as a KEM in oqs provider? I tried to edit the generate.yml file there but I didnt succeed.

Any help or hibt is appreciated!

**Comment by seschu** [comment]
see my other discussion https://github.com/open-quantum-safe/oqs-provider/discussions/330#discussion-6051962 (sorry for spamming ...)

**Reply by baentsch** [reply]
See answer there. What about deleting this discussion then?
